test/e2e/performanceprofile/functests/1_performance/performance.go

```diff
@@ -290,15 +290,54 @@ var _ = Describe("[rfe_id:27368][performance]", Ordered, func() {
 	})
 
 	Context("RPS configuration", func() {
-		It("[test_id:55012] Should have the correct RPS configuration", func() {
+		BeforeEach(func() {
 			if profile.Spec.CPU == nil || profile.Spec.CPU.Reserved == nil {
 				Skip("Test Skipped due nil Reserved cpus")
 			}
+		})
+		It("[test_id: 59572] Check RPS Mask is applied to atleast one single rx queue on all veth interface", func() {
+			if profile.Spec.WorkloadHints != nil && profile.Spec.WorkloadHints.RealTime != nil &&
+				!*profile.Spec.WorkloadHints.RealTime && !profileutil.IsRpsEnabled(profile) {
+				Skip("realTime Workload Hints is not enabled")
+			}
+			count := 0
+			expectedRPSCPUs, err := cpuset.Parse(string(*profile.Spec.CPU.Reserved))
+			Expect(err).ToNot(HaveOccurred())
+			var vethInterfaces = []string{}
+			for _, node := range workerRTNodes {
+				allInterfaces, err := nodes.GetNodeInterfaces(node)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(allInterfaces).ToNot(BeNil())
+				// collect all vethinterfaces in a list
+				for _, iface := range allInterfaces {
+					if iface.Bridge == true && iface.Physical == false {
```

Comment:
```iface.Bridge == true```? I believe you mean ```false```

Reply:
@Tal-or no it should be true as veth interfaces are part bridge ovs-system.